src/AlgoliaSearch.js

```diff
@@ -653,6 +653,52 @@ AlgoliaSearch.prototype.searchUserIDs = function(data, callback) {
   });
 };
 
+/**
+ * Set strategy for personalization
+ *
+ * @param {Object} data
+ * @param {Object} data.eventsScoring Scoring to apply to events
+ * @param {Object} data.facetsScoring Scoring to apply on facets
+ * @return {Promise|undefined} Returns a promise if no callback given
+ * @example
+ * client.setStrategy({
+ *   eventsScoring: {
+ *      "Add to cart": { score: 50, type: "conversion" },
+ *      Purchase: { score: 100, type: "conversion" }
+ *   },
+ *   facetsScoring: {
+ *      brand: { score: 100 },
+ *      categories: { score: 10 }
+ *   }
+ * });
+ */
+AlgoliaSearch.prototype.setStrategy = function(data, callback) {
```

Comment:
will sync with API clients teams on a better naming for this function.

Reply:
I confirm this should be `setPersonalizationStrategy` and `getPersonalizationStrategy`. Thank you for bringing this up 🙏